packages/cryptobox/src/main/CryptoboxSession.ts

```diff
@@ -26,10 +26,10 @@ class CryptoboxSession {
   public pk_store: CryptoboxCRUDStore;
   public session: ProteusSession.Session;
 
-  constructor(id: string, pk_store: CryptoboxCRUDStore, session: ProteusSession.Session) {
-    this.id = id;
+  constructor(id: string | undefined, pk_store: CryptoboxCRUDStore, session: ProteusSession.Session) {
```

Comment:
Shouldn't we set `id` as last parameter if it's optional?

Reply:
Yes, we should but I didn't want to break the API.

Reply:
But I will do a refactoring. Otherwise this will never change. 😎  👍

Reply:
It gave me some very nice ideas so I refactored it. Thanks Florian!!